straxen/plugins/peaklets/peaklet_classification.py

```diff
@@ -105,14 +105,11 @@ def compute(self, peaklets):
         is_s2[is_large_s1 | is_small_s1] = False
         ptype[is_s2] = 2
 
-        return dict(
```

Comment:
Actually what is the motivation we want to change here?

Reply:
The motivation is that I use `super.compute()` in PeakletClassifcationSOM and it is easier to work with a returned structure array than a dictionary. This style change has no impact on the remaining straxen workflow.